sqlalchemy_exasol/base.py

```diff
@@ -692,8 +693,52 @@ def visit_big_integer(self, type_, **kw):
     def visit_large_binary(self, type_):
         return self.visit_BLOB(type_)
 
-    def visit_datetime(self, type_):
-        return self.visit_TIMESTAMP(type_)
+    # --- Date/time ---
+
+    # Some SQLAlchemy versions dispatch DateTime() to 'DATETIME' (upper-case)
+    def visit_DATETIME(self, type_, **kw):
```

Comment:
Is it intentional that they aren't built referring to one another? For instance, we could have visit_DATETIME as it currently is defined and then have visit_datetime call self.visit_DATETIME and pass on the values.